dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WCCustomProperties.java

```diff
@@ -530,7 +530,7 @@ public static void setCustomPropertyVariables() {
 
         ERROR_PAGE_COMPATIBILITY = customProps.getProperty("com.ibm.ws.webcontainer.contenttypecompatibility");
         MAP_FILTERS_TO_ASTERICK = Boolean.valueOf(customProps.getProperty("com.ibm.ws.webcontainer.mapfilterstoasterisk")).booleanValue();
-        SUPPRESS_HTML_RECURSIVE_ERROR_OUTPUT = Boolean.valueOf(customProps.getProperty("com.ibm.ws.webcontainer.suppresshtmlrecursiveerroroutput")).booleanValue();
+        //SUPPRESS_HTML_RECURSIVE_ERROR_OUTPUT = Boolean.valueOf(customProps.getProperty("com.ibm.ws.webcontainer.suppresshtmlrecursiveerroroutput")).booleanValue(); // issue 14242
```

Comment:
Should we also comment out this line?

public static boolean SUPPRESS_HTML_RECURSIVE_ERROR_OUTPUT;

This probably will also require a bundle version update I believe with feature checker since we're removing a public variable.

Thoughts?

Reply:
I think we can just leave it there for the future record.

Reply:
It's a public variable and this is exposed as an SPI so I think we should at the very least comment it out.

I would however expect feature checker to blow up removing a public variable from an SPI so if it doesn't we'll need to talk with the tooling team again since I think you bumped the version already since they last updated the baselines.